### DISCUSSION TITLE
Dec 2023 Review - PR2: Quadratic Models and Meanings

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#44] [Show and tell]
Section: https://teambasedinquirylearning.github.io/precalculus/2024/PR2.html

Author: @kathypinzon 

Primary Reviewer: @sheppardw  (feedback by Dec 11 morning)

**Comment by AbbyANoble** [comment]
Need to remove the "Activities go here..." at the top!

**Reply by AbbyANoble** [reply]
Or change to an observation that introduces the section.

**Reply by sheppardw** [reply]
Yes, need to remove the line below the first Activity number:
1.2.1 Activities
Remove this line: Activities go here! Don’t forget to put text in <p> tags or it won’t show up.

**Comment by AbbyANoble** [comment]
To help them explain why it's not linear in [Activity 1.2.1(b)](https://teambasedinquirylearning.github.io/precalculus/2024/PR2.html#task-13) , you may instead ask them to find the slope between various pairs of values. "Pick any two points in the table above and find the slope between them. Repeat for another (or two more?) sets. What do you notice?"

**Comment by sheppardw** [comment]
For the function in the first activity, since you are using meters, Are you just rounding up for the first coefficient since it should be 9.8/2? We usually use feet for these problems so that we can use 32/2=16. I am happy to say we are rounding it.

**Reply by siwelwerd** [reply]
Hmmm.  That might be worth an `<aside>` mentioning that we have rounded the gravitational constant.

**Reply by kathypinzon** [reply]
changed to feet and readjusted values

**Comment by sheppardw** [comment]
First of all, I like your questions and the way you designed the multiple choice.  For Activity 1.2.3. , the function is a downward facing parabola, which is not very realistic for population growth.  What about just making the quadratic with a positive leading coefficient and the vertex could agree with the starting year or the starting year could be somewhere to the right of the vertex?

**Reply by sheppardw** [reply]
If you do change the quadratic, though, you may need to move the increasing/decreasing question to the first activity. Then again, you could have the starting year for measuring the population be to the left of the vertex and the problem would be solved. :-) Not trying to make it more complicated haha, just thinking of what my students might say or ask!

**Reply by kathypinzon** [reply]
I would say that many small towns have downward facing quadratic growth.  They had a boom from some industry, which has now gone under and so people are leaving the town. I think it's ok to leave as downward parabola.

**Comment by sheppardw** [comment]
For Activity 1.2.4.  It looks good!!  

Some suggestions:

The unit price of an item affects its' supply and demand. That is, if the unit price **goes up,** 

I would change "goes up" to "increases". Also I think an apostrophe should be added to the end of "its".
 
I think you could add a question asking about the maximum revenue.

**Comment by sheppardw** [comment]
Ok my final comment is that I think this would be a good place to introduce a rectangle with perimeter and area application problem! Ask for them to create the function and then find the maximum area. Unless we do that somewhere else?  But I think an application problem where students have to build their function is good prep for Calculus, etc.  
Good work!!
